sim/src/kernels/computeEta.cl

```diff
@@ -4,64 +4,87 @@
 #ifndef __OPENCL_VERSION__
 #define __kernel
```

Comment:
Why are these here? Should this file be compiled at all if we are not using OpenCL?